src/components/NibrsHistogram.js

```diff
@@ -5,10 +5,24 @@ import React from 'react'
 
 import NibrsHistogramDetails from './NibrsHistogramDetails'
 import XAxis from './XAxis'
+import { slugify } from '../util/text'
 
 class NibrsHistogram extends React.Component {
   state = { hover: null }
 
+  componentDidMount() {
+    document.addEventListener('click', this.handleClick)
```

Comment:
i'm a little wary of attaching a clickhandler to the entire document (i'm worried about unintended consequences of that but maybe that's irrational) -- what if we limited to clicks within the nibrs card?

to be honest, i'm also ok leaving it as is (resetting when you stop hovering) though i know that's not quite how it works in our other charts (trend, agency), but those don't really have a default text state